### ISSUE TITLE
Will the training and testing code for downstream tasks be made publicly available

### ISSUE THREAD

**Issue by yuliang-liang** [WeixiongLin/PMC-CLIP#12]
Hi, thank you for your sharing!
Will the training and testing code for downstream tasks be made publicly available？This will be really helpful for us to follow and 
make comparison with your work.

**Comment by WeixiongLin** [comment]
Thanks for your interest in our work. We would not provide the code for downstream tasks, since the main purpose is to provide pretrained image and text encoders. While maintaining a project about downstream tasks takes far more efforts.

You can finetune the encoders on your own tasks, and I am glad to help with that if any problem occurs.

**Comment by KebinWu** [comment]
For the finetuning on image classification task, could you share more details? 

 I suppose that we need to add a new classification head, how is the new head constructed? besides,  shall we have the same learning rates for the clip model & head?  

thank you.

**Comment by WeixiongLin** [comment]
The new prediction head is just a Fully Connected layer actually. You can adjust the learning rate considering your own situation. As far as I know, the hyper-params such as learning rate, num of training epochs is not the determinants of model performance.

**Comment by WeixiongLin** [comment]
I will close the issue for now,  feel free to reopen it if you have further questions.
